.env.example

```
@@ -2,3 +2,15 @@
 SHELVE_TOKEN=
+
+# --- eve agent (agent/) ---
+# Model routing via the Vercel AI Gateway (skip if `vercel link` already
+# gives you a VERCEL_OIDC_TOKEN locally).
+AI_GATEWAY_API_KEY=
+# Vercel Connect GitHub connector slug (create it in the Vercel dashboard,
+# then `vercel link` + `vercel env pull` so the agent can mint tokens).
+EVE_GITHUB_CONNECTOR=github/personal
+# Shared secret that unlocks the portfolio's /chat page. Leave unset to keep
+# it closed in production (local dev keeps working without it).
+EVE_CHAT_USERNAME=hugo
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EVE_CHAT_USERNAME key should go before the EVE_GITHUB_CONNECTOR key

(UnorderedKey)